pkg/cmd/server/origin/authorizer.go

```diff
@@ -40,18 +57,19 @@ func NewAuthorizer(informers InformerAccess, projectRequestDenyMessage string) a
 	)
 	nodeAuthorizer := node.NewAuthorizer(graph, nodeidentifier.NewDefaultNodeIdentifier(), kbootstrappolicy.NodeRules())
 
-	openshiftAuthorizer := authorizerunion.New(
+	return unionFilter(
```

Comment:
rather than do this, why not make a "NoOpinion" authorizer that is the default value for the `maybeDenyAuthorizer`?